src/shared/components/dataAccessTokens/DataAccessTokensDropdown.tsx

```diff
@@ -49,6 +49,8 @@ export class DataAccessTokensDropdown extends React.Component<
     }
 
     @computed get getDatDropdownList(): any {
+        const supported_dat_methods = ['oauth2', 'uuid'];
```

Comment:
maybe put this in src/shared/constants.ts to avoid the duplication

Reply:
of course! ahaha just updated